tomcat/src/main/java/org/apache/coyote/http11/HttpSession.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.coyote.http11;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class HttpSession {
+
+    public static final String SESSION_TYPE = "JSESSIONID";
+
+    private final String id;
+    private final Map<String, Object> values = new HashMap<>();
+
+    public HttpSession() {
+        this.id = String.valueOf(UUID.randomUUID());
```

Comment:
```suggestion
        this.id = String.valueOf(UUID.randomUUID());
        this.id = UUID.randomUUID().toString();
```

요거는 정말 사소한 제 생각인데, UUID는 null값이 들어오는 경우가 없어서 `toString()` 을 사용하는 방법도 직관적일 것 같아요!